app/views/events/partials/_calendar_long_events.html.erb

```diff
@@ -0,0 +1,12 @@
+<h4><%= t('events.long') %></h4>
+<ul>
+  <% events.each do |event| %>
+    <li>
+      <%= link_to event do %>
+        <%= event.title %>
+        -
+        <%= l event.start, format: :long if event.start %> - <%= l event.end, format: :long if event.end %>
```

Comment:
Might be better to have something like `[l(event.start, format: :long), l(event.end, format: :long)].compact.join(' - ')`
to avoid a random `-` when only one date is present.

Or you could use the `neatly_printed_date_range` helper method.